src/threadpool.c

```diff
@@ -261,6 +268,62 @@ static void init_once(void) {
   init_threads();
 }
 
+int uv_set_threadpool_size(unsigned int number) {
+  unsigned int spin_nthreads;
+
+  /* Ensure the threadpool is initialized */
+  if (nthreads == 0)
+    uv_once(&once, init_once);
```

Comment:
Since it's a once handle you can call it every time.

Reply:
With that one. Idk how’s gonna init the thread pool first. I just  used uv_once on all possibilities -just in case-

Reply:
You actually have to call it first and foremost, then get the mutex, then you can check the value of `nthreads`. It's not safe otherwise.